forum/qa-include/pages/message.php

```diff
@@ -68,8 +68,13 @@
 
 //	Check the user exists and work out what can and can't be set (if not using single sign-on)
 
-	if ( !qa_opt('allow_private_messages') || !is_array($toaccount) || ($toaccount['flags'] & QA_USER_FLAGS_NO_MESSAGES) )
-		return include QA_INCLUDE_DIR.'qa-page-not-found.php';
+        if (!qa_opt('allow_private_messages') || !is_array($toaccount) || ($toaccount['flags'] & QA_USER_FLAGS_NO_MESSAGES)) {
+
+            if(qa_get_logged_in_level() < QA_USER_LEVEL_EDITOR) {
+                return include QA_INCLUDE_DIR.'qa-page-not-found.php';

```

Comment:
wygląda to strasznie podejrzanie. Powinno się includować plik i pod nim return "OK" lub true lub cokolwiek. A samo includowanie powinno być najlepiej w bloku try..catch

Reply:
Jeśli wywale ten return, i zostanie samo include, nie pojawia się strona błędu - widać białą stronę, więc to jest wymagana instrukcja.